dependencies/classpath.txt

```diff
@@ -1,41 +1,42 @@
-androidx.databinding:databinding-common:7.1.1
-androidx.databinding:databinding-compiler-common:7.1.1
-com.android.databinding:baseLibrary:7.1.1
-com.android.tools.analytics-library:crash:30.1.1
-com.android.tools.analytics-library:protos:30.1.1
-com.android.tools.analytics-library:shared:30.1.1
-com.android.tools.analytics-library:tracker:30.1.1
+androidx.databinding:databinding-common:7.2.0
```

Comment:
Currently working on fixes for all these leaking runtime classpath dependencies.